### ISSUE TITLE
I can't see all cpu cores

### ISSUE THREAD

**Issue by bugraduz** [xma/T480-Clover#39]
ı can't see all cpu cores in stat menus why?
![Ekran Resmi 2020-12-18 13 07 49](https://user-images.githubusercontent.com/39997863/102602069-10be2880-4132-11eb-89d7-479198ef8e58.png)

**Comment by xma** [comment]
same for me. 

real core count is ok anyway in 'about this mac'/'System report'/'Hardware'

Does previous issue https://github.com/xma/T480-Clover/issues/34  related to headphone is fixed ? :=)

**Comment by bugraduz** [comment]
yes ı fixed previous two issues. But why don't work all cpu core sensors? Could the problem be caused by YOGASMC?
